client/resources/version.properties

```diff
@@ -0,0 +1,4 @@
+project.version=${project.version}
+project.name=${project.name}
+project.auxiliary_flag=
+project.java.version=${maven.compiler.source}
```

Comment:
The ADR says that this field is used for "Version of the language used for executing the SDK". Can you confirm?

Reply:
I asked myself same kind of question...
My reading of "language_version" was "language version" which is currently very static 😕 (but in the future we could imaging releasing a multiple release JAR enabling to use java11 native http client)
And this is why I added the JVM related information which is very dynamic...
@650elx do you have some feedback ?

Reply:
`language_version` purpose is to gather statistical data about what `language versions` customers are using.
For example; PythonSDK supports version 3.8, 3.9, 3.10, 3.11 - one of those will end up in header.

Node JS?
Let's say we support two versions; 18LTS and 20LTS - one of those will end up in header.

Why we need that information?
To know what's import for customers so we can make better business decisions.

Same with `implementation type` or `auxiliary_flag`

Reply:
Ok, thank you.
For java we cannot have access to the "language versions customers are using"
We can have access to :
- what was used during compilation for Sinch Java SDK (this is the project.java.version=${maven.compiler.source} I'm sending to header
- JVM version used for execution

None of these are related to client java version they are using.
Even JVM version is not related to language version: you can compile for java 11 and use a java 17 JVM instance.

So, @asein-sinch , in current state, I think current header is fine; and in the future if we support multi release jar, then, the java version could have more values

Reply:
@JPPortier So maybe `JVM version used for execution` is what we're looking for?

@Dovchik How the thing looks like on the .NET side?

Reply:
> @JPPortier So maybe `JVM version used for execution` is what we're looking for?

@650elx This why these information are present within `auxilliary_flag`
From sample above; `...; AdoptOpenJDK,14.0.2)`

Reply:
It,s straightforward version of NET
Monickers like: net5, net6, net7, net4.8, netcore3

Reply:
@asein-sinch @650elx 
Updated according to last synch: 
- the `language_version` is based onto `JVM version`
- `auxilliary_flag` contains the  `JVM name`

Example: `User-Agent: sinch-sdk/0.0.1-SNAPSHOT (Java/17.0.8; Apache; Eclipse Adoptium)`